tests/beaker/tests/create_lv_with_param_alloc.rb

```diff
@@ -0,0 +1,82 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96574 - create logical volume without parameter 'alloc'"
+
+#initilize
+pv = '/dev/sdc'
+vg = ("VolumeGroup_" + SecureRandom.hex(2))
+lv = [("LogicalVolume_" + SecureRandom.hex(3)), ("LogicalVolume_" + SecureRandom.hex(3)), \
+("LogicalVolume_" + SecureRandom.hex(3)), ("LogicalVolume_" + SecureRandom.hex(3)), ("LogicalVolume_" + SecureRandom.hex(3))]
+
+# Teardown
+teardown do
+  confine_block(:except, :roles => %w{master dashboard database}) do
+    agents.each do |agent|
+      remove_all(agent, pv, vg, lv)
+    end
+  end
+end
+
+pp = <<-MANIFEST
+physical_volume {'#{pv}':
+  ensure  => present,
+}
+->
+volume_group {'#{vg}':
+  ensure            => present,
+  physical_volumes  => '#{pv}',
+}
+->
+logical_volume{'#{lv[0]}':
+  ensure        => present,
+  volume_group  => '#{vg}',
+  alloc         => 'anywhere',
+  size          => '20M',
+}
+->
+logical_volume{'#{lv[1]}':
+  ensure        => present,
+  volume_group  => '#{vg}',
+  alloc         => 'contiguous',
+  size          => '10M',
+}
+->
+logical_volume{'#{lv[2]}':
+  ensure        => present,
+  volume_group  => '#{vg}',
+  alloc         => 'cling',
+  size          => '15M',
+}
+->
+logical_volume{'#{lv[3]}':
+  ensure        => present,
+  volume_group  => '#{vg}',
+  alloc         => 'inherit',
+  size          => '30M',
+}
+->
+logical_volume{'#{lv[4]}':
+  ensure        => present,
+  volume_group  => '#{vg}',
+  alloc         => 'normal',
+  size          => '5M',
+}
+MANIFEST
+
+step 'Inject "site.pp" on Master'
+site_pp = create_site_pp(master, :manifest => pp)
+inject_site_pp(master, get_site_pp_path(master), site_pp)
+
+step 'Run Puppet Agent to create logical volumes'
+confine_block(:except, :roles => %w{master dashboard database}) do
```

Comment:
You've used ```%w{master dashboard database}``` twice in this test. If you assign the value to a variable it will protect against fat finger mistakes.